src/payment/strategies/stripev3/stripev3-payment-strategy.ts

```diff
@@ -243,40 +214,45 @@ export default class StripeV3PaymentStrategy implements PaymentStrategy {
                     billing_details: billingDetails,
                 });
             }
+        }
+    }
 
-            case StripeElementType.iDEAL: {
-                const data = this._mapStripePaymentData(StripePaymentMethodType.iDEAL, returnUrl);
+    private async _executeWithVaulted(payment: OrderPaymentRequestBody, token: string, shouldSetAsDefaultInstrument: boolean | undefined): Promise<InternalCheckoutSelectors> {
+        let formattedPayload: { [key: string]: unknown };
 
-                return await this._getStripeJs().confirmIdealPayment(clientSecret, data, { handleActions: false });
-            }
+        formattedPayload = {
```

Comment:
Just one thing why don't we assign this in line 221?

Reply:
Good catch!